python/docs/source/_templates/spark_footer.html

```diff
@@ -0,0 +1,3 @@
+<p class="copyright">
+    {{copyright}} The Apache Software Foundation, Licensed under the <a href="https://www.apache.org/licenses/LICENSE-2.0">Apache License, Version 2.0</a>.
```

Comment:
FYI: I follow this copyright format from [Apache Spark official web page](https://spark.apache.org/).